templates/bundles/PrestaShopBundle/Admin/Configure/AdvancedParameters/ImportDataConfiguration/Blocks/import_data_configuration.html.twig

```diff
@@ -32,15 +32,13 @@
   </h3>
   <div class="card-block row">
     <div class="card-text">
-      <div class="row">
-        <div class="col">
+
           <div class="alert alert-info" role="alert">
             <p class="alert-text">
               {{ 'Please match each column of your source file to one of the destination columns.'|trans }}
             </p>
           </div>
```

Comment:
It looks like the indentation should be changed, also check every other changes if indentation is fine